ios/Exponent/Versioned/Core/UniversalModules/EXNotifications/EXScopedNotificationsUtils.m

```diff
@@ -20,17 +20,48 @@ + (BOOL)shouldNotification:(UNNotification *)notification beHandledByExperience:
 
 + (NSString *)scopedCategoryIdentifierWithId:(NSString *)categoryId forExperience:(NSString *)experienceId
 {
-  NSString *escapedExperienceId = [EXScopedNotificationsUtils escapedString:experienceId];
+  NSString *scope = [EXScopedNotificationsUtils escapedString:experienceId];
   NSString *escapedCategoryId = [EXScopedNotificationsUtils escapedString:categoryId];
-  return [NSString stringWithFormat:@"%@/%@", escapedExperienceId, escapedCategoryId];
+  return [NSString stringWithFormat:@"%@//%@", scope, escapedCategoryId];
```

Comment:
this is just making it more complicated -- what if the scope key ends with `/`? just use `/` as a delimiter and escape it normally, we don't need to do anything creative here